mesonbuild/dependencies/base.py

```diff
@@ -1450,7 +1450,10 @@ def _detect_dep(self, name: str, modules: T.List[T.Tuple[str, bool]], components
                     cfgs = [x for x in tgt.properties['IMPORTED_CONFIGURATIONS'] if x]
                     cfg = cfgs[0]
 
-                is_debug = self.env.coredata.get_builtin_option('debug');
+                is_debug = self.env.coredata.get_builtin_option('buildtype') == 'debug'
```

Comment:
Shouldn't this still be `'debug'` instead of `'buildtype'`?

Reply:
also, this would currently miss `debugoptimized`

Reply:
No, like I said in my commit message, `debugoptimized` uses the release CRT unless you override with `b_vscrt`. This is documented: https://mesonbuild.com/Builtin-options.html#b_vscrt-from_buildtype

This code is identical to what we do when detecting the Qt dependency and elsewhere.